README.zh_CN.md

```diff
@@ -0,0 +1,169 @@
+  ![Image](misc/image/4.png?raw=true)
+5. 点击 "Publish repository" 将你刚才做的更改上传到 GitHub 。在本案例，你不需要提前在 GitHub 上创建仓库，GitHub Desktop 将帮你自动完成。(译者注： GitHub Desktop 更新后默认将仓库设为私有，需要在 Publish repository 界面将 Keep this code private 这个勾选去掉)
+  ![Image](misc/image/6.png?raw=true)
+
+#### 为了开始在此牌组工作，你的协作者需要：
+1. 在他们的电脑安装 [Github Desktop](https://desktop.github.com/) 。
+2. 克隆你已经创建的仓库。他们可以通过在 GitHub 仓库页面点击 "Clone or download -> Open in Desktop" 做到这一点。
+	![Image](misc/image/7.png?raw=true)
+3. [导入牌组](#导入).
+
+如果有些人**只是想使用**你在 GitHub 上上传的**牌组**，他们可以[直接从仓库导入牌组]()。
+
+#### 如何上传更改
+
+当你或你的一个协作者想将你们做出的更改上传到 GitHub 时，你们需要：
+
+1. 通过点击 "Fetch Origin" 从 GitHub 获取其他人的提交历史，如果有新的更改，则可以点击 "Pull Origin" 获取最新的更改。
+  ![Image](misc/image/8.png?raw=true)
+2. 为了合并你和其他人做出的更改，请[导入牌组](#导入)。
+3. 导出牌组到你仓库的本地目录，然后导出将覆盖 media 文件夹和 json 文件。(替代选项是，你可以将牌组导出到其他地方，然后将 media 文件夹和 json 文件复制到仓库目录下，然后选择合并同名文件夹，替换已存在的文件。)
+4. 在评论区添加 comment ，点击 "Commit to master" 上传你做出的更改。
+  ![Image](misc/image/4.png?raw=true)
+5. 点击 "Push Origin" 上传你做出的更改到 GitHub 。
+
+如果你只想**获取别人做出的最新更改**，你只需要执行第一步和第二步。
+
+### 命令行工作流程
+#### 协作准备
+4. 在你的电脑上安装 git 。
+5. 到导出的目录。
+6. 用下面的命令初始化仓库：
+   
+    ```
+    git init
+    git remote add origin git@github.com:<username>/<repository>.git
+    ```
+
+    <username> 是你的 GitHub 用户名 (我的是 Stvad) ， <repository> 是仓库的名字 (DeckX)。 所以本案例中的命令应该是：
+
+    ```
+    git remote add origin git@github.com:Stvad/DeckX.git
+    ```
+7. 添加目录下的所有内容到仓库：
+
+    ```
+    git add *
+    git commit -m "initial export"
+    ```
+8. 上传你做出的更改到 GitHub：
+
+    ```
+    git push origin master
+    ```
+
+#### 为了开始在此牌组工作，你的协作者需要：
+
+1. 在他们的电脑室安装 git
+2. 克隆你创建好的仓库：
+
+    ```
+    git clone https://github.com/Stvad/DeckX.git
+    ```
+
+3. [导入牌组](#导入).
+
+如果有些人**只是想使用**你在 GitHub 上上传的**牌组**，他们可以[直接从仓库导入牌组]()。
+
+#### 如何上传更改
+
+当你或你的一个协作者想将你们做出的更改上传到 GitHub 时，你们需要：
+
+1. 从 GitHub 获取最新的更改：
+   
+    ```
+    git pull
+    ```
+2. 为了合并你和其他人做出的更改，请[导入牌组](#导入)。
+3. 导出牌组到你仓库的本地目录，然后导出将覆盖 media 文件夹和 json 文件。(替代选项是，你可以将牌组导出到其他地方，然后将 media 文件夹和 json 文件复制到仓库目录下，然后选择合并同名文件夹，替换已存在的文件。)
+4. 将更改添加到仓库：
+
+    ```
+    git add *
+    git commit -m "new updates"
+    ```
+5. 上传你做出的更改到 GitHub ：
+
+    ```
+    git push origin master
+    ```
+
+如果你只想**获取别人做出的最新更改**，你只需要执行第一步和第二步。
+
+
+## 通用协作工作流程
+目前的工作流程可以表述为：
+* 用户创建或者导入一个 Anki 牌组。
+* 用户做了一些更改(比如笔记内容、牌组设置、牌组结构或笔记模板)。
+* 然后用户以 JSON 格式导出牌组(附带媒体文件夹和该牌组中引用的媒体文件)并分享给其他用户。例如通过为此创建 GitHub 仓库。
+* 其他人可以直接修改 JSON 文件，也可以导入牌组到他们自己的 Anki 中并做一些修改。
+* 然后这些人做出的更改就会更新最初的 JSON 文件(必要时将几个更改合并在一起)。
+* 之后，最初的用户（和其他人）可以导入更新的牌组，将这些新的更改整合到他们的集合中。
+
+## 导出
+导出请到菜单：文件>导出
+
+选择牌组(**注**：不支持导出所有的牌组，你需要选择一个特定的牌组)并将导出格式改为 "CrowdAnki JSON representation (*directory )" 。
+点击导出之后，选择一个保存导出的目录。
+
+## 导入
+导入请到菜单：文件>"CrowdAnki: Import from disk" 并选择牌组储存的目录。 
+
+## 从 GitHub 导入
+从 GitHub 导入牌组请到菜单：文件>"CrowdAnki: Import from Github" 并以建议的格式输入 GitHub 用户名和仓库名。
+
+以我的仓库为例，为了获取我的[Ankigaokao-理科-理综-生物-分子与细胞](https://github.com/L-M-Sherlock/Ankigaokao-ScienceComprehensive-Biology-MoleculesAndCells)，你需要输入
```

Comment:
Why change this from pointing to the git deck?

Reply:
Did you mean why I changed the deck in the case? Because I think the git deck is in English , it may confuse chinese people.
Maybe I should translate your git deck to chinese? I hope I have spare time to do it.